pkgs/by-name/tk/tkdiff/package.nix

```diff
@@ -0,0 +1,40 @@
+{ diffutils, fetchurl, lib, makeBinaryWrapper, stdenv, tk, unzip }:
+
+stdenv.mkDerivation {
+  pname = "tkdiff";
+  version = "5.6-r189";
+
+  src = fetchurl {
+    # 5.6 + fix to make "tkdiff /dev/null file" work again.
```

Comment:
Only from the version number I would have guessed that this is a stable release. Can we maybe download the stable release and apply the fix as a patch?

Reply:
thanks! I changed version -> `5.6-patch-189`, do you have a better suggestion? added the patch to this change to patch the stable version.

Reply:
should be good enough

Reply:
I think we can avoid the suffix. The patch is just  a bugfix, and we don't usually modify the version because of those.

Reply:
updated and squashed commits.